acorn/acorn-tests.ts

```diff
@@ -67,3 +67,6 @@ acorn.getLineInfo('string', 56);
 
 acorn.plugins['test'] = function (p: acorn.Parser, config: any) {
 }
+
+acorn.tokenizer('console.log("hello world)', {locations: true}).getToken();
+acorn.tokenizer('console.log("hello world)', {locations: true})[Symbol.iterator]().next();
```

Comment:
A more interesting test for [Symbol.iterator] could be to try `for ... of` over the result to ensure that protocol can be used correctly by TS.